ibm/service/kubernetes/resource_ibm_container_cluster.go

```diff
@@ -1066,38 +1066,38 @@ func resourceIBMContainerClusterUpdate(d *schema.ResourceData, meta interface{})
 		d.Set("force_delete_storage", forceDeleteStorage)
 	}
 
-	if d.HasChange("default_pool_size") && !d.IsNewResource() {
-		workerPoolsAPI := csClient.WorkerPools()
-		workerPools, err := workerPoolsAPI.ListWorkerPools(clusterID, targetEnv)
-		if err != nil {
-			return err
-		}
-		var poolName string
-		var poolContains bool
-
-		if len(workerPools) > 0 && workerPoolContains(workerPools, defaultWorkerPool) {
-			poolName = defaultWorkerPool
-
-			poolContains = true
-		} else if len(workerPools) > 0 && workerPoolContains(workerPools, computeWorkerPool) && workerPoolContains(workerPools, gatewayWorkerpool) {
-			poolName = computeWorkerPool
-			poolContains = true
-		}
-		if poolContains {
-			poolSize := d.Get("default_pool_size").(int)
-			err = workerPoolsAPI.ResizeWorkerPool(clusterID, poolName, poolSize, targetEnv)
-			if err != nil {
-				return fmt.Errorf("[ERROR] Error updating the default_pool_size %d: %s", poolSize, err)
-			}
-
-			_, err = WaitForWorkerAvailable(d, meta, targetEnv)
-			if err != nil {
-				return fmt.Errorf("[ERROR] Error waiting for workers of cluster (%s) to become ready: %s", d.Id(), err)
-			}
-		} else {
-			return fmt.Errorf("[ERROR] The default worker pool does not exist. Use ibm_container_worker_pool and ibm_container_worker_pool_zone attachment resources to make changes to your cluster, such as adding zones, adding worker nodes, or updating worker nodes")
-		}
-	}
+	// if d.HasChange("default_pool_size") && !d.IsNewResource() {
```

Comment:
Commenting out this block of code will prevent customers from managing the number of workers in their default worker pool, right?

Do we want to stage the delivery of this? First release, just update the `ForceNew`. Second release, remove management of the default worker pool from the cluster resource. Will this allow for a smoother transition for customers?